packages/core/src/errors/RequestError/index.ts

```diff
@@ -57,4 +63,23 @@ export default class RequestError extends Error {
 
     return conditional(this.data instanceof ZodError && formatZodError(this.data).join('\n'));
   }
+
+  /**
+   * Parse the error body with i18n context
+   */
+  toBody(ctx: WithI18nContext): RequestErrorBody {
```

Comment:
pass an i18next instance instead?

Reply:
if a body requires i18n, better to remove `get body() {}`?

Reply:
updated